### ISSUE TITLE
[Accessibility]  Task settings menu cannot be opened with keyboard

### ISSUE THREAD

**Issue by HadiAyoub** [ShareX/ShareX#5810]
![image](https://user-images.githubusercontent.com/11969612/132780207-1ef9cbcf-bb03-4a24-b640-2610e75223bb.png)

This menu, even when focused, cannot be opened via keyboard. Must be clicked with mouse.

Renders shareX unusable by those who are navigating via keyboard only.
